koala/Spreadsheet.py

```diff
@@ -300,8 +302,7 @@ def eval_volatiles_from_ast(self, ast, node, cell):
         results = []
         context = cell["sheet"]
 
-        if (node.token.tvalue == "INDEX" and node.parent(ast) is not None and node.parent(ast).tvalue == ':') or \
-            (node.token.tvalue == "OFFSET"):
+        if (node.token.tvalue == "INDEX" or node.token.tvalue == "OFFSET"):
```

Comment:
I know but this kind of pattern is in several parts of the code (not only in `Spreadsheet.py`, and under different forms.
I deliberately chose to leave it like that to gain time.
We should open a specific issue.